qa/L0_state_management/check_results.py

```diff
@@ -117,7 +117,7 @@ def check_early_exit(self):
         return True
 
     def check_continue_after_checkpoint(self,
-                                        expected_resnet_count=3,
+                                        expected_resnet_count=4,
```

Comment:
This makes the comment of line 137 incorrect? Can you explain how you now guarantee that no measurements are taken before the interrupt? I would have expected changes in the `test.sh` for this test were needed

Reply:
Yes, as mentioned in the previous comment, the profiling of the second model will not be started.

After the `interrupt_handling` test case is finished, only the first model will be profiled with its two configurations. Then, in the test case `check_continue_after_checkpoint`, the remaining two models will be profiled, with two configurations each. Therefore, we need to verify a total of four entries for the first to models.

This makes the comment on line 137 incorrect. I need to modify the comment, sorry for missing doing so.

I think to fix this script failure, we can make modifications to either `test.sh` or `check_results.py`. I have chosen to modify `check_results.py`. Please let me know if you would like `test.sh` to be modified instead.

Reply:
Please update the comment on line 137